plugins/tiddlywiki/katex/config.tid

```diff
@@ -0,0 +1,13 @@
+title: $:/plugins/tiddlywiki/katex/config
+
+<$list filter="[all[shadows+tiddlers]tag[$:/tags/katex/config]!has[draft.of]]">
```

Comment:
For consistency, the tag should be `$:/tags/KaTeX/Config`

Reply:
Ok I will fix this.